cdhweb/pages/management/commands/exodus.py

```diff
@@ -353,5 +430,17 @@ def get_media_files(self):
     def get_wagtail_image(self, image):
         # get the migrated wagtail image for a foreign-key image
         # using image file basename, which is migrated as image title
-        return Image.objects.get(title=os.path.basename(image.name))
-
+        try:
+            return Image.objects.get(title=os.path.basename(image.name))
+        except Image.DoesNotExist as err:
+            print("Error locating image %s: %s" % (image, err))
+            return None
+
+    def person_image_exodus(self):
+        # for people with profile, set larger image as wagtail image for person
+        for profile in Profile.objects.filter(user__person__isnull=False):
+            # NOTE not sure why Profile.user.person fails; works in console?
+            person = Person.objects.get(user=profile.user)
+            if profile.image:
```

Comment:
Could maybe filter on image in the queryset? (Although I think I had problems with a similar filter somewhere, not sure if it's worth it)

Do we have any cases where someone has a thumbnail but no larger image, maybe for people who are only on list views? Should we use the thumbnail as a fallback if image is not set?

Reply:
I was running into cases where people didn't have an image when I expected them to; maybe that's why...I'm not sure, but I bet that does happen. Probably we should use the thumbnail as a fallback, yep

Reply:
I counted 57 cases of this, actually, with:
```py
Profile.objects.filter(image="").exclude(thumb="").count()
```
added a check to use thumbnails as a fallback.